test/attr/Inputs/DeprecatedSymbolsModule.swift

```diff
@@ -28,8 +28,8 @@ public protocol ImprotedNonDeprecatedProtocol {
     var a: String { get }
 }
 
-// expected-warning {{'importedDeprecatedType' is deprecated}}
-// expected-warning {{'importedDeprecatedProtocol' is deprecated}}
+// expected-warning @-2 {{'importedDeprecatedType' is deprecated}}
+// expected-warning @-1 {{'importedDeprecatedProtocol' is deprecated}}
```

Comment:
These should be `@+2` and `@+1` if you want them to refer to the line where the `extension` is

Reply:
Fixed